src/tekore/_client/api/playlist/items.py

```diff
@@ -139,3 +141,69 @@ def playlist_remove(
         return self._generic_playlist_remove(
             playlist_id, {"tracks": items}, snapshot_id
         )
+
+    @scopes([scope.playlist_modify_public], [scope.playlist_modify_private])
+    @send_and_process(top_item("snapshot_id"))
+    def playlist_remove_occurrences(
+        self,
+        playlist_id: str,
+        refs: List[Tuple[str, int]],
+        snapshot_id: Optional[str] = None,
+    ) -> str:
+        """
+        Remove items by URI and position.
+
+        .. warning::
+        This feature is undocumented and may stop working at any time.
+
+        Parameters
+        ----------
+        playlist_id
+            playlist ID
+        refs
+            a list of tuples containing the URI and index of items to remove
+        snapshot_id
+            snapshot ID for the playlist
+        Returns
+        -------
+        str
+            snapshot ID for the playlist
+        """
+        gathered = {}
+        for uri, ix in refs:
+            gathered.setdefault(uri, []).append(ix)
+
+        items = [
+            {"uri": uri, "positions": ix_list} for uri, ix_list in gathered.items()
+        ]
+        return self._generic_playlist_remove(
+            playlist_id, {"tracks": items}, snapshot_id
+        )
+
+    @scopes([scope.playlist_modify_public], [scope.playlist_modify_private])
+    @send_and_process(top_item("snapshot_id"))
+    def playlist_remove_indices(
```

Comment:
I tried running the tests locally, and this still fails for me. Did you find an alternative?

Reply:
This has worked for me when tried to use this method in my app. I have not tried the test though, but I will.